### PR TITLE
DIGITAL AGENCY

### PR DESCRIPTION
**Pull request by Gbengacode**
### I made the following changes to this work

- I added HTML markups
- I added CSS style
- I added javascript functionalities
- I added images

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+### Prerequisites
+
+- Have git install on your local machine
+
+### Setup
+
+- Clone the repository to your machine
+
+- Run on your browser
```
Comment (dasileker) [inline README.md:33]: - [ ] Be specific about this part, how you go to the Github page and how to download and how to clone etc...

File: about.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  </head>
+  <body>
+  
+    <div class="small-menu">
+      <div class="icon"><i class="fa fa-times fa-2x menu-icon"></i></div> 
+       <ul>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Contact</a></li>
+        <li><a href="#">Request a Quest</a></li>
+       </ul>
```
Comment (dasileker) [inline about.html:25]: - [ ] Replace the about link with the home page link I couldn't return back.

cccc

File: about.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+              <li><img src="./assets/img/partners/mozilla.svg" alt="mozilall logo"></li>
+            </ul>
+        </div>
+     </section>
+     
+    </main>
+      <footer class="footer">
+        <div class="footer-container">
+          <div class="img-container">
+            <img src="./assets/img/logo/logo.png" alt="logo">
```
Comment (dasileker) [inline about.html:112]: - [ ] The image logo should be close to the text in the center of the footer.

- Your design

BBBB

- Behance design

mmm

File: index.html [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+            
+            </ul>
+          </nav>
+        </div>
+      
+        <div class="main-nav">
+          <div class="logo"><a href=""><img src="./assets/img/logo/logo.png" alt="logo"/></a></div>
+          <nav>
+            <ul>
+              <li class="bar"><a href="#"><img src="./assets//img/logo/hamburger.svg" alt="bar"></a></li>
```
Comment (dasileker) [inline index.html:46]: - [ ] The hamburger mobile menu has a button on the left side remove it, please.

dddd

File: index.html [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+            <i class="fab fa-searchengin"></i>
+            <h3>Seo</h3>
+            <p>We make your site friendly to search engines</p>
+          </div>
+
+          <div class="small-card">
+            <i class="far fa-business-time"></i>
+            <h3>Ui / Ux design</h3>
+            <p>We provide good ui and ux design for your product</p>
+          </div>
```
Comment (dasileker) [inline index.html:92]: - [ ] The card text and logos should be in the center.

- your design

aaaa


- Behance design

vvvv

File: index.html [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+      </section>
+      
+    </main>
+      <footer class="footer">
+        <div class="footer-container">
+          <div class="img-container">
+            <img src="./assets/img/logo/logo.png" alt="logo">
+          </div>
+       
+          <p> Volvo Team </br> All right reserved @2021</p>
```
Comment (dasileker) [inline index.html:128]: - [ ] The same thong for the footer the logo should be close to the centered text.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Gbengacode,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @Gbengacode, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**